content/en/docs/alphashape.md

```diff
@@ -6,28 +6,118 @@ sections:
 
-##### 1.1. Points from the network
 
-![](/images/docs/technical_documentation/alphashape/set_points.webp)
+#### Routing Modes
 
-##### 1.2. Convex Hull 
-Imagine you take an a rubber band and you stretch it around the points. This is called convex hull. As you can see this representation is missing level of detail but comes with the advantage of fast computation. 
+Depending on the chosen travel mode GOAT uses different parts of the street network. To keep the platform flexible the street network categories can be adjusted to the local study context when setting up the tool.
 
-![](/images/docs/technical_documentation/alphashape/convex_hull.webp)
+Depending on the availability of data in the study area and the planning purpose different routing profiles are available. These are walking, cycling, pedelec and public transport.
 
-##### 1.3. Concave Hull
-A concave hull usually begins with a convex hull as the starting geometry and then tries to better fit the point dataset. Depending on the chosen concavity the shape can be more or less precise. Different parameters can also be used for the calculation of the isochrones, depending on the needed precision. However, rising precision comes with higher computation times. 
 
-![](/images/docs/technical_documentation/alphashape/concave_hull.webp)
+#### 1. Walking
 
-Therefore, GOAT uses a specific parameter for the calculation of the isochrones, that represents a compromise between accuracy and computation time. Thanks to the [PLV8 extension](https://github.com/plv8/plv8) the [Concaveman](https://github.com/mapbox/concaveman) library can be used directly in the PostgreSQL database.
+Default routing profile for walking, considering all paths accessible by foot.
+
+Default walking speed: 5km/h
+
+<img src="\images\docs\technical_documentation\alphashape\walking_en.webp" alt="walking isochrone" style="max-height:85px;"/>
+
+#### 2. Cycling
+
+#### 2.1 Standard
+
+Default routing profile for cycling, considering all paths accessible by bicycle.
+
+Default cycling speed: 15km/h Depending on the surface, smoothness and slope of the different street segments, the cycling speed is adjusted accordingly.
+
+<img src="\images\docs\technical_documentation\alphashape\standard_en.webp" alt="standard isochrone" style="max-height:88px;"/>
+
+#### 2.2 Pedelec
+
+Same as the standard routing profile, but with an increased cycling speed of 23 km/h.
+For Pedelecs, slopes are considered with a lower impedance than for Standard bicycles.
+
+<img src="\images\docs\technical_documentation\alphashape\pedelec_en.webp" alt="pedelec isochrone" style="max-height:80px;"/>
+
+
+#### 3. Transit
+
+In this routing mode users can compute intermodal accessibility centered around public transport.
+As you can be seen in the figure users can adjust: 
```

Comment:
"How the user changes the stations" is wrong. It should be "How the user exists the station"